testgrid/specs/os-firstlast.yaml

```diff
@@ -49,4 +49,8 @@
   version: "20.04"
   vmimageuri: https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-amd64.img
   preinit: ""
-
+- id: ubuntu-2204
+  name: Ubuntu
+  version: "22.04"
+  vmimageuri: https://cloud-images.ubuntu.com/jammy/current/jammy-server-cloudimg-amd64.img
+  preinit: ""
```

Comment:
add this to os-full too please?

Reply:
It's already [there](https://github.com/replicatedhq/kURL/blob/main/testgrid/specs/os-full.yaml#L67-L71).